plugins/scaffolder/api-report.md

```diff
@@ -56,10 +58,8 @@ import { z } from 'zod';
 // @public @deprecated (undocumented)
 export const createScaffolderFieldExtension: typeof createScaffolderFieldExtension_2;
 
-// @public
-export function createScaffolderLayout<TInputProps = unknown>(
-  options: LayoutOptions,
-): Extension<LayoutComponent<TInputProps>>;
+// @public @deprecated (undocumented)
```

Comment:
We should document these deprecations in the changesets and suggest how to fix them :)

Reply:
Maybe a seperate changeset entirely where you can provide examples similar to the ones for #15399